plugins/lookup/onepassword_ssh_key.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2023, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+    name: onepassword_ssh_key
+    author:
+      - Mohammed Babelly (@mohammedbabelly20)
+    requirements:
+      - C(op) 1Password command line utility version 2 or later.
+    short_description: Fetch SSH Keys stored in 1Password
+    version_added: "10.3.0"
+    description:
+      - P(community.general.onepassword_ssh_key#lookup) wraps C(op) command line utility to fetch ssh keys from 1Password.
+    notes:
+      - By default, it returns the private key value in PKCS#8 format, unless O(ssh_format=true) is passed.
+      - The pluging works only for C(SSHKEY) type items.
+      - This plugin requires C(op) version 2 or later.
+
+    options:
+      _terms:
+        description: Identifier(s) (case-insensitive UUID or name) of item(s) to retrieve.
+        required: true
+        type: list
+        elements: string
+      ssh_format:
+        description: Output key in SSH format if true. Otherwise, outputs in the default format.
+        required: false
+        default: false
+        type: bool
+
+    extends_documentation_fragment:
+      - community.general.onepassword
+      - community.general.onepassword.lookup
+"""
+
+EXAMPLES = """
+- name: Retrieve the private key of ssh key from 1Password
+  ansible.builtin.debug:
+    var: lookup('community.general.onepassword_ssh_key', 'SSH Key', ssh_format=true)
```

Comment:
Generally `msg` should be preferred over `var`:
```suggestion
    msg: "{{ lookup('community.general.onepassword_ssh_key', 'SSH Key', ssh_format=true) }}"
```